### PR TITLE
proofreading 2d/canvas_layers.rst : change passive voice to direct voice

### LINKED ISSUE #4206: Edit 2d/canvas_layers.rst
The CanvasLayer page uses a lot of passive voice while our guideline is to favor the direct voice, and the explanations really aren't as clear as they could be. The page needs edits.

### PR DESCRIPTION
**Pull request by pyacier**
Fixes #4206
At least I hope it fixes the passive voice issue.

@NathanLovato or someone else can you point me what explanation could be clearer ?

### CODE REVIEW

File: tutorials/2d/canvas_layers.rst [hunk 1]
```diff
@@ -6,34 +6,34 @@ Canvas layers
...
-:ref:`Control <class_Control>` both inherit from
-:ref:`CanvasItem <class_CanvasItem>`, which is the base for all 2D
-nodes. CanvasItems can be arranged in trees. Each item will inherit
-its parent's transform. This means that when the parent is moved, the children
+:ref:`CanvasItem <class_CanvasItem>` is the base for all 2D nodes,
+regular 2D nodes, such as :ref:`Node2D <class_Node2D>` or
+:ref:`Control <class_Control>` both inherit from :ref:`CanvasItem <class_CanvasItem>`.
+CanvasItems can be arranged in trees. Each item will inherit
+its parent's transform. This means that when the parent move, the children
 will move too.
```
Comment (NathanLovato) [inline tutorials/2d/canvas_layers.rst:14]:
```suggestion
:ref:`CanvasItem <class_CanvasItem>` is the base for all 2D nodes, be it regular
     2D nodes, such as :ref:`Node2D <class_Node2D>`, or `Control
     <class_Control>`. Both inherit from :ref:`CanvasItem <class_CanvasItem>`.
     You can arrange canvas items in trees. Each item will inherit its parent's
     transform: when the parent moves, its children move too.
```

File: tutorials/2d/canvas_layers.rst [hunk 2]
```diff
@@ -6,34 +6,34 @@ Canvas layers
...
 
 CanvasItem nodes, and nodes inheriting from them, are direct or indirect children of a
-:ref:`Viewport <class_Viewport>`, and will be displayed through it.
+:ref:`Viewport <class_Viewport>`, that will display them.
```
Comment (NathanLovato) [inline tutorials/2d/canvas_layers.rst:17]:
```suggestion
:ref:`Viewport <class_Viewport>`, that display them.
```

File: tutorials/2d/canvas_layers.rst [hunk 3]
```diff
@@ -6,34 +6,34 @@ Canvas layers
...
 
-A Viewport has the property
+The Viewport's property
 :ref:`Viewport.canvas_transform <class_Viewport_property_canvas_transform>`,
-which allows applying a custom
-:ref:`Transform2D <class_Transform2D>` transform to the CanvasItem hierarchy it contains. Nodes such as
-:ref:`Camera2D <class_Camera2D>` work by changing that transform.
+allows to apply a custom :ref:`Transform2D <class_Transform2D>`
+transform to the CanvasItem hierarchy it contains. Nodes such as
+:ref:`Camera2D <class_Camera2D>` change that transform to work.
```
Comment (NathanLovato) [inline tutorials/2d/canvas_layers.rst:23]:
```suggestion
:ref:`Camera2D <class_Camera2D>` work by changing that transform.
```

The original phrasing sounds clearer to me.

File: tutorials/2d/canvas_layers.rst [hunk 4]
```diff
@@ -6,34 +6,34 @@ Canvas layers
...
 
-Effects like scrolling are best achieved by manipulating the canvas transform property. This approach is more
-efficient than moving the root canvas item (and hence the whole scene).
+Approaches to achieve effects like scrolling that manipulate the canvas transform property are
+more efficient than those that move the root canvas item (and hence the whole scene).
```
Comment (NathanLovato) [inline tutorials/2d/canvas_layers.rst:26]:
```suggestion
To achieve effects like scrolling, manipulating the canvas transform property is
more efficient than moving the root canvas item and the entire scene with it.
```

File: tutorials/2d/canvas_layers.rst [hunk 5]
```diff
@@ -6,34 +6,34 @@ Canvas layers
...
 
 Usually though, we don't want *everything* in the game or app to be subject to the canvas
 transform. Examples of this are:
 
 -  **Parallax Backgrounds**: Backgrounds that move slower than the rest
    of the stage.
--  **UI**: Think of a user interface (UI) or Heads-up display (HUD) superimposed on our view of the game world. We want a life counter, score display and other elements to retain their screen positions even when our view of the game world is changing.
+-  **UI**: Think of a user interface (UI) or Heads-up display (HUD) superimposed on our view of the game world. We want a life counter, score display and other elements to retain their screen positions even when our view of the game world change.
```
Comment (NathanLovato) [inline tutorials/2d/canvas_layers.rst:33]:
```suggestion
-  **UI**: Think of a user interface (UI) or Heads-up display (HUD) superimposed on our view of the game world. We want a life counter, score display and other elements to retain their screen positions even when our view of the game world changes.
```

File: tutorials/2d/canvas_layers.rst [hunk 6]
```diff
@@ -47,7 +47,7 @@ CanvasLayers also have their own transform and do not depend on the
 transform of other layers. This allows the UI to be fixed in screen-space
 while our view on the game world changes.
 
-An example of this is creating a parallax background. This can be done
+An example of this is the creation of a parallax background. This can be done
```
Comment (NathanLovato) [inline tutorials/2d/canvas_layers.rst:50]:
```suggestion
An example of this is creating a parallax background. This can be done
```

**Review by NathanLovato — COMMENTED** [review]
The -ing form or noun describing actions is often more natural than expressions. E.g. "Creating something" over "The creation of something". Also, sometimes, grammatically, you need an -ing form after the preposition `to`. I'm unsure in this case but my grammar corrector says `Approaches to achieving ...` is the correct form.

**Comment by NathanLovato** [comment]
Thanks for your contribution, I made edits directly here: https://github.com/godotengine/godot-docs/pull/4327/commits/ae51070767ce84c0d160607a6ddae02a31d16371
Will merge when the CI says OK

**Comment by pyacier** [comment]
Sorry to have you do the work in the end. English is not my native language, I have some progress to make.

**Comment by NathanLovato** [comment]
No worries, it's completely normal. I make edits directly mostly because it's more efficient than leaving you comments for the both of us. And you get to see the changes for your next contribution.

I'm not native either, it all comes with practice! (I still make mistakes in English too)

**Pull request merged**